CONTRIBUTING.md

```diff
@@ -13,7 +13,8 @@ $ composer update
 
 In addition to installing project dependencies, Composer will check that the
 required extension version is installed. Directions for installing the extension
-may be found [here](https://php.net/manual/en/mongodb.installation.php).
+may be found [here](https://php.net/manual/en/mongodb.installation.php). Composer will also install the submodule required for
+running spec tests.
```

Comment:
```suggestion
may be found [here](https://php.net/manual/en/mongodb.installation.php).
Composer will also install the submodule required for running spec tests.
```

Consistent wrapping.

Reply:
Lost track of this. I was also wondering why I didn't notice this, until I realised that PhpStorm had the link target collapsed, suggesting that the text will fit into the 80-character limit we impose 🤦‍♂️